tasks/main.yml

```diff
@@ -1 +1,31 @@
 # role tasks
+
+- name: Install necessary packages
+  package:
+    name: parted
+    state: present
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+
+- name: Clear the disk signatures
```

Comment:
I would change the name of this so that the user knows we're running the `wipefs` command.

Reply:
updated

Reply:
updated